terraform/keycloak-sandbox/iam.tf

```diff
@@ -0,0 +1,63 @@
+resource "aws_iam_role" "keycloak_ecs_execution" {
+  name               = "keycloak_ecs_execution_role_${var.environment}"
+  assume_role_policy = data.aws_iam_policy_document.ecs_assume_role.json
+
+  tags = merge(
+    local.common_tags,
+    map(
+      "Name", "api-ecs-execution-iam-role-${var.environment}",
+    )
+  )
+}
+
+resource "aws_iam_role" "keycloak_ecs_task" {
+  name               = "keycloak_ecs_task_role_${var.environment}"
+  assume_role_policy = data.aws_iam_policy_document.ecs_assume_role.json
+
+  tags = merge(
+    local.common_tags,
+    map(
+      "Name", "api-ecs-task-iam-role-${var.environment}",
+    )
+  )
+}
+
+data "aws_iam_policy_document" "ecs_assume_role" {
```

Comment:
Generally policies are include as separate json.tpl files

Reply:
Is there a major advantage to that, especially when it's a short policy? I don't really mind, but we've currently got a mix of Terraform and JSON definitions (including the place I copied this policy from):

https://github.com/nationalarchives/tdr-terraform-environments/search?q=statement
https://github.com/nationalarchives/tdr-terraform-backend/search?q=statement

Reply:
For small policies not that much difference. Its more a consistency thing. It was something Paul S suggested to do when he was here.

Reply:
I've move the policy into a template file. 👍